File: app/src/main/java/lemond/annoying/gamerscompanion/fragment_now/fragment_page/view/HypedPageFragment.java
package lemond.annoying.gamerscompanion.fragment_now.fragment_page.view;

import lemond.annoying.gamerscompanion.R;
import lemond.annoying.gamerscompanion.app.GlideApp;
import lemond.annoying.gamerscompanion.fragment_now.adapter.GameGridAdapter;


public class HypedPageFragment extends NowPageFragment {

    public HypedPageFragment() {}

    public static HypedPageFragment newInstance() {
        return new HypedPageFragment();
    }

    @Override
    public void initializeGridAdapter() {
        gameGridAdapter = new GameGridAdapter(GlideApp.with(this), getString(R.string.more_hyped));
    }

}

File: app/src/main/java/lemond/annoying/gamerscompanion/main_activity/fragment_now/fragment_page/view/HypedPageFragment.kt
package lemond.annoying.gamerscompanion.main_activity.fragment_now.fragment_page.view

import lemond.annoying.gamerscompanion.R
import lemond.annoying.gamerscompanion.app.GlideApp
import lemond.annoying.gamerscompanion.main_activity.fragment_now.adapter.GameGridAdapter


class HypedPageFragment : NowPageFragment() {

    override fun initializeGridAdapter() {
        gameGridAdapter = GameGridAdapter(GlideApp.with(this), getString(R.string.more_hyped))
    }

    companion object {
        fun newInstance(): HypedPageFragment {
            return HypedPageFragment()
        }
    }

}
